content/learn/manuals.md

```diff
@@ -11,31 +11,36 @@ layout: "manuals"
   <span class="mwl"><a href="/grass{{< currentVersionNodots.inline  >}}{{- .Site.Data.grass.current_version_nodots -}}{{</currentVersionNodots.inline >}}
 /manuals/index.html " target="_blank"> GRASS GIS {{< grassVersion version="current" type="short">}}  (current) </a></span>
   <a href="/grass{{< currentVersionNodots.inline  />}}/manuals/index.html" class="inl btn btn-primary" target="_blank">View HTML</a> <a href="/grass{{< currentVersionNodots.inline  />}}/manuals/grass-{{< currentVersionShort.inline  >}}{{- .Site.Data.grass.current_version_short -}}{{</currentVersionShort.inline >}}_html_manual.zip" class="inl btn btn-secondary">Download ZIP</a>
-  </li>
-   <li>
+ </li>
+ <li>
   <span class="mwl"><a href="/grass{{< legacyVersionNodots.inline  >}}{{- .Site.Data.grass.legacy_version_nodots -}}{{</legacyVersionNodots.inline >}}/manuals/index.html " target="_blank"> GRASS GIS {{< grassVersion version="legacy" type="short">}}  (legacy) </a></span>
   <a href="/grass{{< legacyVersionNodots.inline  />}}/manuals/index.html" class="inl btn btn-primary" target="_blank">View HTML</a>
    <a href="/grass{{< legacyVersionNodots.inline  />}}/manuals/grass-{{< legacyVersionShort.inline  >}}{{- .Site.Data.grass.legacy_version_short -}}{{</legacyVersionShort.inline >}}_html_manual.zip" class="inl btn btn-secondary">Download ZIP</a>
-  </li>
+ </li>
+ <li>
+  <span class="mwl"><a href="/grass{{< previewVersionNodots.inline  >}}{{- .Site.Data.grass.preview_version_nodots -}}{{</previewVersionNodots.inline >}}/manuals/index.html " target="_blank"> GRASS GIS {{< grassVersion version="preview" type="short">}}  (preview) </a></span>
+  <a href="/grass{{< previewVersionNodots.inline  />}}/manuals/index.html" class="inl btn btn-primary" target="_blank">View HTML</a>
+   <a href="/grass{{< previewVersionNodots.inline  />}}/manuals/grass-{{< previewVersionShort.inline  >}}{{- .Site.Data.grass.preview_version_short -}}{{</previewVersionShort.inline >}}_html_manual.zip" class="inl btn btn-secondary">Download ZIP</a>
+ </li>
  <!--
```

Comment:
all of this is commented out, should we keep it?

Reply:
I have removed it in 6cd3b5fdd7553095b55beddf23f6d352cdfa92ef